app/views/directors/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<% @directors.each do |director| %>
+  <%= link_to director.name, movies_path(director_id: director.id) %>
+<% end %>
```

Comment:
Why is there added functionality for  a directors index here and in the director's controller?

Reply:
I hit an issue where when I navigated to the movie index, there were no movies. That prompted me to go to the movies_controller, where I saw that saw that there was branch logic checking for a Director ID, and that the movies that were passed to the index were director specific. I stopped reading there...and subsequently rewrote the test to navigate to a movie via a link in the director's page and built out the functionality. Only after that did I realize that the movie-less index page was because, when building out my test, the visit movies_path was positioned before my movie objects (a copy and paste error). Moreover, I realized that the index did not require a director_id, but had some logic to display a director's movies if an director_id was present, or all movies if director_id was absent. By that time, I thought it best to continue on the path because it still seemed to check the box in terms of completing all user stories. In hindsight, even if it were the case that movies were only displayed on an index view if a director_id was present, I think I could have merely passed it as a parameter on 'visit movies_path' and saved myself the trouble.